contracts/BorrowerOperations.sol

```diff
@@ -203,6 +221,8 @@ contract BorrowerOperations is
         vars.price = priceFeed.fetchPrice();
         bool isRecoveryMode = _checkRecoveryMode(vars.price);
 
+        if(isRecoveryMode) recoveryModeMutexHandler(true);
```

Comment:
just the best practice if no objections
```suggestion
        if(isRecoveryMode) {
            await recoveryModeMutexHandler(true);
        }
```

Reply:
updated in this [commit](https://github.com/DistributedCollective/zero-contracts/pull/7/commits/87907c748eda0da187bd5f5ed5f48f5d4c511b4e)